homeassistant/components/recorder/models.py

```diff
@@ -474,7 +474,7 @@ class StatisticsMeta(Base):  # type: ignore[misc,valid-type]
     )
     __tablename__ = TABLE_STATISTICS_META
     id = Column(Integer, Identity(), primary_key=True)
-    statistic_id = Column(String(255), index=True)
+    statistic_id = Column(String(255), index=True, unique=True)
```

Comment:
Why would we have an `id` and `statistic_id` column? can't we merge them?

Reply:
The id here is an integer, the statistic I'd is typically the entity_id

Reply:
Since it's a small table, it's fine to have a string as a primary key I think?

Reply:
No, it's not fine.
The `id` here is used to map to the `id` in the `statistics` and `statistics_short_term` tables which are NOT small and where duplicating the statistic_id would make the tables many times larger.